index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Montserrat Cruz | Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="header-nav">
+    <a href="#" id="mylogo">My Logo</a>
+    <nav class="navMenu">
+  
+        
+        <div id="menu">
+            <i class="bar"></i>
+            <i class="bar"></i>
+            <i class="bar"></i>
+        </div>
+         
+        
+      </nav>
+
+  </header>
+
+  <main class="flex-container wrapper">
+    <section id="head-container">
+      <h1 class="headers">
+        I'm Montserrat, <br />
+        Glad to see you!
+      </h1>
+      <p class="description">
+        I’m a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience. If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me!
+      </p>
+      <h2 class="connect">LET’S CONNECT</h2>
+
+      <ul class="social-media-icons">
+        <li>
+          <a href="https://twitter.com/montsecruzg9" rel="noopener noreferrer" target="_blank">
+            <img src="./assets/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/montserrat-cruz-695594120/" rel="noopener noreferrer" target="_blank">
+            <img src="assets/Linkedin.png" alt="linkedIn" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="assets/medium.png" alt="medium" />
+          </a>
+        </li>
+        <li>
+          <a href="https://github.com/montsecruz9" rel="noopener noreferrer" target="_blank">
+            <img src="assets/github.png" alt="git" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="assets/angel.png" alt="angel" />
+          </a>
+        </li>
+      </ul>
+    </section>
```

Comment:
You did great work on implementing the changes required by the previous reviewer, but there are some changes that still need to fix to make your project complete.

Kindly I suggest you stick to the Figma design as much as possible to make your project look more professional.

> ## The Figma Design
>
><img width="387" alt="Screenshot 2023-06-03 at 7 16 15 PM" src="https://github.com/montsecruz9/personal-portfolio/assets/91237525/ed78a671-ea3c-41d4-9a8e-429960487d7d">
>
> ## Your Design
>
><img width="373" alt="Screenshot 2023-06-03 at 7 12 19 PM" src="https://github.com/montsecruz9/personal-portfolio/assets/91237525/8460fccc-54d0-46cd-b908-8be4fa42eaba">

Reply:
@stanleySimeon Hi stanley! Thanks for your comments. I do have the "Let's Connect" text exactly as the figma design is showing it. Look: 
![image](https://github.com/montsecruz9/personal-portfolio/assets/104812189/09f2154c-379d-4820-96a3-2afb005590b3)